tests/tests/chat/matrixChatArea.spec.ts

```diff
@@ -166,6 +166,13 @@ test.describe("matrix chat area property @matrix", () => {
 
     await expect(page.getByTestId("closeChatButton")).toBeVisible();
 
+    try{
+      await page.getByTestId("closeModal").click();
+    }
+    catch{
+      console.log("no modal to close")
+    }
+
```

Comment:
You should use a handler instead.

See https://github.com/thecodingmachine/workadventure/blob/ff849286d3354466d2a2b316a7bd1b07cceeef72/tests/tests/mobile/oidc.spec.ts#L22-L22